src/js/app.js

```diff
@@ -91,6 +91,49 @@ $(document).ready(function() {
     }
   });
 
+  //Add toggle effect for sliding #menu
+  let showingMenu = false;
+  const $menu = $('#menu');
+  const $main = $('main');
+  const config2 = {
+    duration: 1000,
+    easing: 'easeOut'
+  };
+  $('#menu-burger,.close-menu,#navigation-links>li>a').on('click', function() {
+    if (showingMenu) {
+      $menu.removeClass('active');
+      $main.removeClass('menu-active');
+      showingMenu = false;
+      event.preventDefault();
+    } else {
+      $menu.addClass('active');
+      $main.addClass('menu-active');
+      showingMenu = true;
+      event.preventDefault();
+    }
+    return showingMenu;
+  });
+
+  const burgerPos_y = $('#menu-burger').offset().top;
+  $(document).on('scroll', function() {
+    let scrollPos_y = $(window).scrollTop();
+    if (scrollPos_y + $(window).width() / 33 > burgerPos_y) {
+      // $('#menu-burger').css('transform', 'translateY(' + (scrollPos_y + ($(window).width() / 33.33) - burgerPos_y) + 'px)');
+      $('#menu-burger').css({
+        'position': 'fixed',
+        'top': '3vw',
+        'left': '3vw',
+        'margin': 0
+      });
+    } else {
+      // $('#menu-burger').css('transform', 'translateY(0)');
+      $('#menu-burger').css({
```

Comment:
That's cute. Yes you can set any CSS in JS and will overwrite any other CSS in the hierarchy (because JS inlines it). One downside is that JS does not prefix, nor minify the CSS. So recommended to add classes instead. In this case nothing will be prefixed, but suppose you'd use `display: flex` in there, you'll get a lot of cross browser issues.